changelogs/fragments/3334-django_manage-split-params.yaml

```
@@ -0,0 +1,2 @@
+bugfixes:
+  - django_manage - parameters ``apps`` and ``fixtures`` should be splitted at whitespaces to keep backward compatibility (https://github.com/ansible-collections/community.general/issues/3333).
```

```suggestion
  - django_manage - parameters ``apps`` and ``fixtures`` are now splitted instead of being used as a single argument (https://github.com/ansible-collections/community.general/issues/3333).
```
Hmm, backwards compatibility probably would require to split at whitespace...